app/tools/android/make_apk.py

```diff
@@ -494,14 +507,12 @@ def MakeApk(options, app_info, manifest):
     else:
       # If the arch option is unspecified, all of available platform APKs
       # will be generated.
-      valid_archs = ['x86', 'x86_64', 'armeabi-v7a']
+      valid_archs = ['x86', 'x86_64', 'armeabi-v7a', 'arm64-v8a']
```

Comment:
Why not use the arch folder names and avoid the need for `ConvertArchFolderToArchName()` to exist altogether?

```python
for arch in AllArchitectures():
  if ConvertArchNameToFolder(arch) in available_archs:
    # yadda, yadda
  else:
    # yadda, yadda
```